docusaurus/docs/guides/index.md

```diff
@@ -3,9 +3,7 @@ title: Overview
 sidebar_position: 10
 ---
 
-## What is a Guide?
-
-A guide is like a recipe. It's a type of documentation that is more advanced and solution-focused than other types. A guide is prescriptive and procedural. For example, "How to deploy a Ziti controller with Helm."
+This section contains operational guidance for specific tasks such as deploying a controller with Helm or as a Linux system service, backing up the controller's database, and migrating controller or router configurations.
```

Comment:
I'd like to keep the "what is a guide" with definition. it sets the stage for the type of documentation located in this section. I could see these new words going along with the former words as they build on the example from the old wording

Reply:
Fine by me. It struck me as superfluous but it might be helpful.

Reply:
conversation marked as resolved, but the change still shows?